package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@snyk/rpm-parser": "^1.1.0",
     "debug": "^4.1.1",
+    "docker-modem": "^2.1.3",
```

Comment:
what do you think about moving to a fixed version?

Reply:
Happy to move to a fixed version.